### PR TITLE
Modified dimensions on the report to make it more readable

### PR DESCRIPTION
**Pull request by gawsalya**
Created a new function: plot_table_small for the smaller tables so height can be adjusted depending on the table. 
Removed sentiment and the number of reviews from the new release table.
Modified the HTML so the first table can fill the first page. 
Changed the number of releases displayed on the first table from 30 to 27 this is for sizing/readability purposes. We will need to implement automatic sizing in the future.

### CODE REVIEW

**Comment by gawsalya** [comment]
Note the last two tables are still pointing to an older date than the current date -- note: @rhKashif mentioned there was a similar problem on the dashboard so please could you either let me know or push the changes in a separate branch. Thanks!

File: report/chart.png
Comment (DMcCallion) [inline report/chart.png:1]: The table is looking really good, but was this meant to be in gitignore?

File: report/lambda_function.py [hunk 1]
```diff
@@ -209,6 +209,35 @@ def get_most_reviewed_release(df_releases: DataFrame) -> str:
     return df_ratings.head(1)["title"][0]
 
 
+def aggregate_release_data_table_one(df_releases: DataFrame) -> DataFrame:
```
Comment (DMcCallion) [inline report/lambda_function.py]: Maybe rather than table one, specify what's happening on that table

File: report/lambda_function.py [hunk 2]
```diff
@@ -293,7 +323,49 @@ def plot_table(df_releases: DataFrame, rows: int) -> Chart:
...
+    ).properties(
+        height=1000,
+        width=1300
+    )
+    return chart
+
+
+def plot_table_small(df_releases: DataFrame, rows: int) -> Chart:
+    """
+    Create a table from a given DataFrame
```
Comment (DMcCallion) [inline report/lambda_function.py]: This is a bit of a vague docstring: what table, what dataframe? - maybe note in the docstrings what columns the table will have?

File: report/lambda_function.py [hunk 3]
```diff
@@ -302,7 +374,9 @@ def plot_table(df_releases: DataFrame, rows: int) -> Chart:
         alt.Text("value", type="nominal"),
     ).properties(
         width=1300,
+        height=200
     )
+    chart.save("chart.png")
```
Comment (DMcCallion) [inline report/lambda_function.py]: Is this save just a test that hasn't been removed or do you later open the file?

**Review by DMcCallion — APPROVED** [review]
Looks good overall, just a few minor nitpicks.

the report looks really good now!

**Comment by rhKashif** [comment]
> Note the last two tables are still pointing to an older date than the current date -- note: @rhKashif mentioned there was a similar problem on the dashboard so please could you either let me know or push the changes in a separate branch. Thanks!

Yep, opened up a pull request on this yesterday which is yet to be reviewed. Not sure how its going to interact with the changes you have made because I have overhauled the `aggregate_release_data` table.

File: report/chart.png
Comment (gawsalya) [reply report/chart.png:1]: yes

**Review by rhKashif — APPROVED** [review]

**Pull request merged**